web/templates/settings/uploaded_files_list.hbs

```diff
@@ -19,15 +19,15 @@
     </td>
     <td class="upload-size" >{{ size_str }}</td>
     <td class="actions">
-        <span class="edit-attachment-buttons">
-            <a type="submit" href="/user_uploads/{{path_id}}" class="btn no-style" title="{{t 'Download file' }}" id="download_attachment" download>
+        <span class="edit-attachment-buttons tippy-zulip-delayed-tooltip" data-tippy-content="{{t 'Download' }}">
```

Comment:
Let's fix the tooltip position for both `Download` and `Delete` options. ![image](https://github.com/zulip/zulip/assets/82862779/06a16971-9dcd-4af8-a1e5-e065ccf26ee9)